testnet/stacks-node/src/neon_node.rs

```diff
@@ -409,15 +400,8 @@ fn try_mine_microblock(
                                 // will need to relay this
                                 next_microblock = Some(microblock);
                             }
-                            Err(NetError::ChainstateError(err_str)) => {
-                                if err_str.contains("NoTransactionsToMine") {
-                                    trace!("Failed to mine microblock because there are no transactions to mine");
-                                } else {
-                                    warn!(
-                                        "Failed to mine one microblock: {:?}",
-                                        &NetError::ChainstateError(err_str)
-                                    );
-                                }
+                            Err(ChainstateError::NoTransactionsToMine) => {
+                                info!("Will keep polling mempool for transactions to include in a microblock");
```

Comment:
Should this be an `info!` log or `debug!` log (or even `trace!` log)? `NoTransactionsToMine` is pretty noisy, and doesn't provide much information other than the fact that the mempool is currently empty.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2234/commits/445e25c5863951ce0606d59b4cf66b08529bf801